salt/utils/templates.py

```diff
@@ -449,7 +449,7 @@ def py(sfn, string=False, **kwargs):  # pylint: disable=C0103
         setattr(mod, kwarg, kwargs[kwarg])
 
     try:
-        data = mod.run()
+        data = mod.run()str0ng
```

Comment:
Yep. Can you tell I didn't test this commit? Reminder to self. No matter how trivial a change. Test it!